src/components/featureHighlight/index.js

```diff
@@ -63,6 +64,10 @@ class FeatureHighlight extends BaseComponent {
      * Message to be displayed
      */
     message: PropTypes.string,
+    /**
+     * The headline to be displayed above the title
+     */
+    headline: PropTypes.string,
```

Comment:
I think this is very specific to what Growth team wants, why not allowing styling the title and the message so thy can achieve the typography they want?

Reply:
Your call. You want me to change it?

Reply:
The point was to control the text design though

Reply:
yea, but it will never good enough, especially if it's an open source component. 
someone will always want to customize it

Reply:
Ok. I'll change it

Reply:
So you think they will never use the three of them together?

Reply:
if the idea was to add a third type of content so people can have 3 different contents then I guess this prop make sense
if we're just adding another content prop with the exact same styling of the `title` prop just so people can render it under the `message` then it's weird, if that's the case it makes more sense to have custom styling over the those content 

Since the current requirement was to still have two contents just in a different order I assume letting them customize the style feels more right